Sources/SwiftDocCUtilities/PreviewServer/RequestHandler/FileRequestHandler.swift

```diff
@@ -67,6 +67,7 @@ struct FileRequestHandler: RequestHandlerFactory {
     /// a list of file and content types allowed in those locations.
     static let assets: [AssetFileMetadata] = [
         AssetFileMetadata(folderPath: "/data/", mimetype: { _ in "application/json" }),
+        AssetFileMetadata(folderPath: "/index/", mimetype: { _ in "application/json" }),
```

Comment:
Is this change related?

Reply:
It was accidentally committed- thanks for catching that! Removed.